.coveragerc

```diff
@@ -22,7 +23,9 @@ cover_pylib = false
 # https://coverage.rtfd.io/en/latest/contexts.html#dynamic-contexts
 # dynamic_context = test_function  # conflicts with `pytest-cov` if set here
 parallel = true
-plugins = Cython.Coverage
+plugins =
```

Comment:
So the PyPy jobs have `sed -i.bak 's/^plugins = .*//g' .coveragerc` which was fine when the (incompatible) `Cython.Coverage` plugin was on the same line. However now that it's on a dedicated line, it just breaks the config.